### ISSUE TITLE
The pre-trained model can't be loaded

### ISSUE THREAD

**Issue by zjgirl** [irfanICMLL/Auto_painter#4]
When I tried loading the pre-trained model, something went wrong like below. It seems that the model's parameters doesn't match the structure? 

Can anyone help? Thank you very much!

> 2020-01-04 20:49:38.604178: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_7/batchnorm/offset/Adam_1 not found in checkpoint
> 2020-01-04 20:49:38.604178: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_7/batchnorm/scale/Adam not found in checkpoint
> 2020-01-04 20:49:38.604221: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_7/conv/filter/Adam not found in checkpoint
> 2020-01-04 20:49:38.604253: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_7/batchnorm/scale/Adam_1 not found in checkpoint
> 2020-01-04 20:49:38.604195: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_7/batchnorm/offset/Adam not found in checkpoint
> 2020-01-04 20:49:38.606036: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_8/batchnorm/offset/Adam_1 not found in checkpoint
> 2020-01-04 20:49:38.606443: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_7/conv/filter/Adam_1 not found in checkpoint
> 2020-01-04 20:49:38.606492: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_8/batchnorm/offset/Adam not found in checkpoint
> 2020-01-04 20:49:38.607934: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_8/batchnorm/scale/Adam not found in checkpoint
> 2020-01-04 20:49:38.612170: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_7/batchnorm/offset/Adam not found in checkpoint
> 	 [[Node: save/RestoreV2_151 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_151/tensor_names, save/RestoreV2_151/shape_and_slices)]]
> 2020-01-04 20:49:38.612178: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_7/batchnorm/offset/Adam not found in checkpoint
> 	 [[Node: save/RestoreV2_151 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_151/tensor_names, save/RestoreV2_151/shape_and_slices)]]
> 2020-01-04 20:49:38.627436: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_7/batchnorm/offset/Adam not found in checkpoint
> 	 [[Node: save/RestoreV2_151 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_151/tensor_names, save/RestoreV2_151/shape_and_slices)]]
> 2020-01-04 20:49:38.627436: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key generator/encoder_7/batchnorm/offset/Adam not found in checkpoint
> 	 [[Node: save/RestoreV2_151 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_151/tensor_names, save/RestoreV2_151/shape_and_slices)]]

**Comment by irfanICMLL** [comment]
The model I provided is a frozen one, which can only be used for demo evaluation.
Sorry for the inconvenient. This project is opened two years ago, and it is very hard for me to find the training ckpt. Luckily, I still have the training images! You can train your own model with the new released data.
